crates/assists/src/handlers/extract_struct_from_enum_variant.rs

```diff
@@ -401,13 +430,19 @@ fn f() {
 }
 "#,
             r#"
+//- /main.rs
 struct V{ pub i: i32, pub j: i32 }
 
 enum E {
     V(V)
 }
 mod foo;
 
+//- /foo.rs
+use crate::E;
```

Comment:
Import is missing?

Reply:
technically this is fine, since this is a child module it sees everything of the parent. That's what the path len check was for, because otherwise this would've generated a `use V;` import which I believe isn't really too useful as it basically doesn't do anything? Though I just realized now that this assist doesn't use the prefixed import lookup, so the imports generated by this ignore the import prefix config so I'm gonna fix that.

Reply:
>technically this is fine, since this is a child module it sees everything of the parent.

huh? This is exactly how Rust modules do **not** work?

https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=bced5b0da2c4a1a27d3ae794352d83e9

Reply:
Oh wait, now why did I think this was how this worked? I'm so sorry 😅 I wonder what I mixed this up with now.